code/modules/mob/living/carbon/human/say.dm

```diff
@@ -297,4 +297,24 @@
 				if(length(cword))
 					rearranged += cword
 			message ="[prefix][jointext(rearranged," ")]"
+		if(oxygen_alert) //Gasping is a mutation, right?
+			var/prefix=copytext(message,1,2)
+			if(prefix == ";")
+				message = copytext(message,2)
+			else if(prefix in list(":","#"))

```

Comment:
doesn't the user also have the ability to set custom prefixes for certain things? Or was it only for languages, ergo it doesn't matter.

Reply:
That's the same code used for tourettes syndrome. It must be the latter